doc/qbittorrent.1

```diff
@@ -1,41 +1,435 @@
-Qt, using the \f[B]libtorrent-rasterbar\f[R] library by Arvid Norberg.
-qBittorrent is similar to uTorrent.
-qBittorrent is fast, stable, light, it supports unicode and it provides
-a good integrated search engine.
-It also comes with UPnP port forwarding / NAT-PMP, encryption (Vuze
-compatible), FAST extension (mainline) and PeX support (utorrent
-compatible).
+\f[I]OPTIONS\f[R] can be passed in from several sources.
+If the same option is specified in multiple sources, the last one takes
+precedence, in the following order:
+.IP "1." 3
+The main configuration file (see the \f[I]FILES\f[R] section)
+.IP "2." 3
+\f[I]ENVIRONMENT\f[R] variables
+.IP "3." 3
+Command-line \f[I]OPTIONS\f[R]
+.PP
+\f[B]IMPORTANT NOTE:\f[R] Settings changed via command-line
+\f[I]OPTIONS\f[R] or \f[I]ENVIRONMENT\f[R] variables not only
+\f[B]override\f[R] their corresponding values in the configuration file,
+they \f[B]also overwrite\f[R] them.
+.PP
+\f[I]ENVIRONMENT\f[R] variables and command-line \f[I]OPTIONS\f[R] can
+currently only override a small subset of all the available settings
+that can be configured in the main configuration file.
+.PP
+qBittorrent can be cleanly terminated by fully closing the GUI
+(including the system tray, if that option is enabled) or by sending the
+\f[B]SIGINT\f[R] or \f[B]SIGTERM\f[R] signals to its process.
 .SH OPTIONS
+.SS Generic program information
+.TP
+\f[B]\f[CB]-h\f[B]\f[R], \f[B]\f[CB]--help\f[B]\f[R]
+Display a short help text about the command-line options and exit.
+.TP
+\f[B]\f[CB]-v\f[B]\f[R], \f[B]\f[CB]--version\f[B]\f[R]
+Display program version and exit.
+.SS General configuration
+.TP
+\f[B]\f[CB]--configuration=\f[B]\f[R]\f[I]NAME\f[R]
+Use a different configuration with name \f[I]NAME\f[R].
+This enables running multiple separate instances of qBittorrent at the
+same time without conflict (provided they don\[cq]t listen on the same
+ports for WebUI access, incoming connections, etc; in this case, the
+additional instances will either fail to start or be functionally
+unusable).
+.RS
+.PP
+Configuration and data files will be stored under a configuration
+hierarchy similar to the default one (see the \f[I]FILES\f[R] section
+for more information) but with a final directory named
+\f[B]qBittorrent_\f[BI]NAME\f[B]\f[R] instead of \f[B]qBittorrent\f[R].
+.PP
+For example, while the main configuration file is stored as
+\f[I]\[ti]/.config/qBittorrent/qBittorrent.conf\f[R], the configuration
+file for a configuration named \f[B]foo\f[R] will be stored as
+\f[I]\[ti]/.config/qBittorrent_\f[BI]foo\f[I]/qBittorrent.conf\f[R].
+.PP
+If a configuration with the given \f[I]NAME\f[R] does not exist yet, it
+is created with default settings.
+If it already exists, it is loaded as a normal configuration would be.
+.PP
+If both \f[B]\f[CB]--configuration\f[B]\f[R] and
+\f[B]\f[CB]--profile\f[B]\f[R] are specified, only
+\f[B]\f[CB]--profile\f[B]\f[R] takes effect.
```

Comment:
I'm not sure you're right here.

Reply:
@glassez I'm quite certain of this. I tested with `$ qbittorrent --profile=qbt_profile_test --configuration=qbt_config_test`. The result was a full configuration under `~/qbt_profile_test`, as predicted (remember that if a relative path is specified in `--profile`, it will be relative to the user's home directory).

Reply:
>I'm quite certain of this.

I'm quite certain of the opposite, after I tested it.
I tested with `qbittorrent --profile=testProfile --configuration=test`. It creates `testProfile/qBittorrent_test` (i.e. it applies both `--profile` and `--configuration` parameters).
>remember that if a relative path is specified in `--profile`, it will be relative to the user's home directory

My result also contradicts this. It is relative to the application's working directory (as the relative path usually behaves).

Reply:
@glassez 

> I'm quite certain of the opposite, after I tested it.
> I tested with `qbittorrent --profile=testProfile --configuration=test`. It creates `testProfile/qBittorrent_test` (i.e. it applies both `--profile` and `--configuration` parameters).

You're right, did not realize they were nested like that...

> My result also contradicts this. It is relative to the application's working directory (as the relative path usually behaves).

Yep, seems I was totally wrong about this.

Reply:
@glassez fixed